doc-content/enterprise-only/openshift/rolemapping-proc.adoc

```diff
@@ -0,0 +1,68 @@
+[id='rolemapping-proc']
+= Providing the LDAP role mapping file
+
+If you configure the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, you must provide a file that defines the role mapping. You must mount this file on all affected deployment configurations.
```

Comment:
Can we avoid the second "you must":
 When you configure the `AUTH_ROLE_MAPPER_ROLES_PROPERTIES` parameter, you must provide a file that defines the role mapping and mount it on all affected deployment configurations.
There is nothing wrong in the current statement, just trying to avoid the second occurrence of "you must" to improve usability.